core/mobile/src/frontend/MobileAuthorizationFrontend.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module OIDC
+ */
+
+import { AccessToken } from "@itwin/core-bentley";
+import { AuthorizationClient } from "@itwin/core-common";
+import { MobileApp } from "./MobileApp";
+
+/** Utility to request OIDC/OAuth tokens from mobile frontend to mobile backend.
+ * @beta
```

Comment:
This should be `@internal` similar to the backend one.

Reply:
Fixed.